src/index.js

```diff
@@ -0,0 +1,41 @@
+import degreeUnit from './degree';
+import fetchImage from './fetchImage';
+
+const display = document.getElementById('display');
+const submit = document.getElementById('submit');
+const weatherAPI = process.env.weatherAPI;
+
+const weatherCall = () => {
+  const location = document.getElementById('location').value;
+  const unit = document.getElementById('unit').value;
+  const weatherData = `http://api.openweathermap.org/data/2.5/weather?q=${location}&APPID=${weatherAPI}`;
+  fetch(weatherData, {
+    mode: 'cors',
+  }).then((data) => data.json()).then((res) => {
+    switch (res.cod) {
+      case 200: fetchImage(res.weather[0].main);
```

Comment:
All this is too coupled so, lets decouple:
- lets have a method that is responsible just  getting the data
- another to prepare and put the data in the UI